lib/Remotery.h

```diff
@@ -220,6 +220,19 @@ typedef struct Sample rmtSample;
 // Handle to the main remotery instance
 typedef struct Remotery Remotery;
 
+// Forward declaration
+struct rmtProperty;
+
+typedef enum rmtSampleType
+{
+    RMT_SampleType_CPU,
+    RMT_SampleType_CUDA,
+    RMT_SampleType_D3D11,
+    RMT_SampleType_OpenGL,
+    RMT_SampleType_Metal,
+    RMT_SampleType_Count,
+} rmtSampleType;
```

Comment:
Missed this when we added support for sample iterators.
This enum is used when iterating the samples.